src/databricks/labs/remorph/bigquery/bigquery.py

```diff
@@ -0,0 +1,18 @@
+import logging
+from typing import ClassVar
+
+from sqlglot.dialects.bigquery import BigQuery
+
+logger = logging.getLogger(__name__)
+
+
+# pylint: disable=function-redefined
+class BigQuery(BigQuery):
```

Comment:
@vijaypavann-db `config.py` also needs modification to use this class

Reply:
Done, Updated `config.py`